contents/teams/analytics-platform/mission.mdx

```diff
@@ -1 +1,8 @@
+# 5-year vision
 
+Our goal is to support customers that are 100x the size of our current largest customers.
+
+This includes making sure that their queries complete in a reasonable amount of time on a very large dataset, but also considers what it is like to use analytics at massive scale in terms of organisation size.
+We want to ensure that such a large org can not only organise their data, but also that they can *trust* data that someone else in a distant part of the org has set up.
```

Comment:
Another way to word this would be these three pillars:
1. Fixing existing issues and making sure we don't regress (e.g. exports, subscriptions, dashboards)
2. Scaling systems to handle greater load (e.g. sessions on events, mat col slots) 
3. New features for large organizations (e.g. schema management)